sites/docs/src/content/docs/tutorials/google_colab/nf-core-colab-guide.md

```diff
@@ -0,0 +1,211 @@
+
+Make sure to also checkout this [blogpost](https://nf-co.re/blog/2025/nf-core-colab-guide) on this topic for more background on running nf-core pipelines using Colab.
+
+## Setting up the environment in Google Colab
+
+If you are new to Google Colab, you can follow the official [Google Colab Getting Started guide](https://research.google.com/colaboratory/faq.html) or the [Colab Welcome Notebook](https://colab.research.google.com/notebooks/intro.ipynb) for instructions on how to register for an account and set up your first notebook.
+
+After opening a Colab notebook with the machine type of your choice, the first step is to install Nextflow and its dependencies. Since Colab does not provide root access, you will need to install Nextflow in your home directory.
+
+### Setting up Java for Nextflow
+
+Before installing Nextflow, you must install Java.
+
+In Colab, run the following commands in a notebook code cell using the `%` and `!` prefixes, or run them directly in your terminal (without the prefixes).
+
+```python
+%cd .. # use '%'before cd to make a permanent change of directories
+!sudo apt update
+!sudo apt install openjdk-17-jdk
+!export JAVA_HOME=/usr/lib/jvm/java-17-openjdk-amd64
+!export PATH=$JAVA_HOME/bin:$PATH
+!source ~/.bashrc
+```
+
+### Installing and Setting up Nextflow
+
+Next, install Nextflow and make it executable by running the following commands in a code cell as before:
+
+```python
+!wget -qO- https://get.nextflow.io | bash # Download Nextflow
+!mv nextflow /usr/bin/nextflow # Move to a path Colab can access
+!chmod +x /usr/bin/nextflow # Make it executable
+!nextflow -v # Test it
+```
+
+### Setting Up Conda for Google Colab
+
+Due to a lack of root access, it is not possible to run nf-core, or any Nextflow pipelines, using the `-profile docker` or `-profile singularity` container based configuration profiles.
+
+That leaves the `conda` profile as the only way to run pipelines.
+
+However, as Google Colab does not natively support conda, you need to install the [condacolab](https://pypi.org/project/condacolab/) package, which will set up Conda for you by running the following commands:
+
+```python
+!pip install -q condacolab
+import condacolab
+condacolab.install()
+```
+
+Make sure to set the conda defaults as follows for smooth pipeline execution:
+
+```python
+!conda config --add channels defaults
+!conda config --add channels bioconda
+!conda config --add channels conda-forge
+!conda config --set channel_priority strict
+```
+
+## Running a test pipeline
+
+...and you're ready to go!
+
+Yes, the setup is that simple!
+
+Now that the main parts of the environment are all set up, we can run the demo pipeline [nf-core/demo](https://nf-co.re/demo/).
+To install the pipeline, run:
+
+```python
+! nextflow pull nf-core/demo
+```
+
+### Preventing Matplotlib Backend Errors in Colab
```

Comment:
OK based on your explanation, I think this is actually very specific to running specific pipelines with specific conda versions of tools - we should remove this section here in the tutorial, but you can keep that in the blog post as it's 'your experience', does that sound OK?